apps/rsa.c

```diff
@@ -226,11 +237,14 @@ int rsa_main(int argc, char **argv)
     }
 
     if (modulus) {
-        const BIGNUM *n;
-        RSA_get0_key(rsa, &n, NULL, NULL);
+        BIGNUM *n;
```

Comment:
Shouldn't this be initialised to NULL?

Reply:
Hmmm, have you found a case where it *has* to?  I don't mind initializing it, but am curious...